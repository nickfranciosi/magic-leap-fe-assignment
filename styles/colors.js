const alert = '#e15021';
const backgroundMain = '#151515';
const border = '#333';
const callout = '#9e4f60';
const callToAction = '#edec51';
const subduedText = '#ddd';
const primaryText = '#fff';

export default {
  alert,
  backgroundMain,
  border,
  callout,
  callToAction,
  subduedText,
  primaryText
};
