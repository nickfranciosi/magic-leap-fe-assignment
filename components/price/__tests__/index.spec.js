/* eslint-env jest */
import React from 'react';
import renderer from 'react-test-renderer';
import 'jest-styled-components';
import Price from '../index';
import colors from '../../../styles/colors';

describe('Price', () => {
  test('it should render', () => {
    const tree = renderer.create(<Price />).toJSON();
    expect(tree).toMatchSnapshot();
  });

  test('it should have dark text on light bg', () => {
    const tree = renderer.create(<Price />).toJSON();
    expect(tree).toHaveStyleRule('color', colors.darkText);
    expect(tree).toHaveStyleRule('background-color', colors.backgroundLight);
  });
});
