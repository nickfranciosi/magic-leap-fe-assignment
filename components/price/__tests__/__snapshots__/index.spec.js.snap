// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Price it should render 1`] = `
.c0 {
  font-size: 32px;
  background-color: #fff;
  width: auto;
  color: #000;
  padding: 16px;
  background: linear-gradient( to right,rgba(255,255,255,1) 26%,rgba(255,255,255,0.04) 70%,rgba(255,255,255,0.02) 71%,rgba(255,255,255,0) 72%,rgba(255,255,255,0) 81% );
}

<h3
  className="c0"
/>
`;
